### PR TITLE
neon/recps: Use vector ops instead of relying on autovec

### PR DESCRIPTION
**Pull request by ngzhian**

### CODE REVIEW

File: simde/arm/neon/recps.h [hunk 1]
```diff
@@ -39,17 +42,7 @@ simde_vrecps_f32(simde_float32x2_t a, simde_float32x2_t b) {
...
-      a_ = simde_float32x2_to_private(a),
-      b_ = simde_float32x2_to_private(b);
-
-    SIMDE_VECTORIZE
-    for (size_t i = 0 ; i < (sizeof(r_.values) / sizeof(r_.values[0])) ; i++) {
-      r_.values[i] = SIMDE_FLOAT32_C(2.0) - (a_.values[i] * b_.values[i]);
-    }
-
-    return simde_float32x2_from_private(r_);
+    return simde_vsub_f32(simde_vdup_n_f32(SIMDE_FLOAT32_C(2.0)), simde_vmul_f32(a, b));
```
Comment (nemequ) [inline simde/arm/neon/recps.h:45]: How about `simde_vmls_f32(simde_vdup_n_f32(SIMDE_FLOAT32_C(2.0)), a, b)` instead?  That way we have a chance to use `_mm_fnmadd_ps` on x86, and it should still degrade to the same series of operations as you have above on other targets.

Comment (ngzhian) [reply simde/arm/neon/recps.h:45]: Good idea, thanks!

**Comment by nemequ** [comment]
Merging the recps stuff; I think the rshrn_n commit was an accidental push?

**Comment by ngzhian** [comment]
Yes oops, forgot to create a new branch :)

**Pull request closed without merging**